package/yast2-country.spec

```diff
@@ -39,13 +39,13 @@ BuildRequires:  yast2-core >= 3.1.12
 # RSpec extensions for YaST
 BuildRequires:  yast2-ruby-bindings >= 3.1.26
 # OSRelease.id
-BuildRequires:  yast2 >= 3.1.211
+BuildRequires:  yast2 >= 3.1.210.2
```

Comment:
Although it doesn't hurt It should continue requiring 3.1.211 version instead of 3.1.210.2. Isn't it?